### PR TITLE
Add Travo and Travo-Jupyter recipes

### PR DESCRIPTION
**Pull request by cmarmo**
Dear conda-forge maintainers,
I would like to maintain the conda recipe for [Travo](https://travo-cr.gitlab.io/travo/).
Travo (from the French *Travaux*) is a lightweight open source Python toolkit automating steps in the assignment workflow for computational practice sessions.

It comes with a simple CLI or Jupyter lab widgets and dashboards: this is why I'm building two subpackages in the recipe. 

Thank you for your attention.

@conda-forge-admin, please ping @conda-forge/help-python

Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [ ] If static libraries are linked in, the license of the static library is packaged (Not applicable).
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by cmarmo** [comment]
After discussion with the other authors, we prefer to have 1.0.0 on conda (if conda maintainers agree to include the package), rather than wit for 1.0.1, I have commented out the test for now and working to a fix for the next bugfix release.
Thanks for your understanding.

File: recipes/travo/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,88 @@
+{% set name = "travo" %}
+{% set version = "1.0.0" %}
+
+package:
+  name: {{ name }}
```
Comment (xhochy) [inline recipes/travo/meta.yaml:5]:
```suggestion
  name: {{ name }}-split
```

File: recipes/travo/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+  sha256: 61da7a967051f93707ba4fe7c13e5399567e9e1eae72ac766dce49e678490f36
+
+build:
+  noarch: python
+  number: 0
+  entry_points:
+    - travo = travo.console_scripts:travo
+    - travo-echo-travo-token = travo.console_scripts:travo_echo_travo_token
+
+test:
```
Comment (xhochy) [inline recipes/travo/meta.yaml:19]: Please move these tests to the outputs

File: recipes/travo/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+          - tqdm
+          - anybadge
+          - i18nice
+          - pyyaml
+
+  - name: travo-jupyter
+    script: build_travo-jupyter.sh
+    requirements:
+        host:
+          - python >=3.8
```
Comment (xhochy) [inline recipes/travo/meta.yaml:52]:
```suggestion
        host:
          - python >=3.8
          - {{ pin_subpackage('travo', exact=True }}
```

File: recipes/travo/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+          - flit-core >=3.2,<4
+          - pip
+        run:
+          - python >=3.8
```
Comment (xhochy) [inline recipes/travo/meta.yaml:56]:
```suggestion
          - python >=3.8
          - {{ pin_subpackage('travo', exact=True }}
```

File: recipes/travo/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,88 @@
...
+  license_file: LICENSE
+  home: https://gitlab.com/travo-cr/travo
+  description: |
+    Travo ([tʁa’vo]) is a lightweight open source Python toolkit that turns your favorite
+    forge into a flexible management solution for computer assignments. It takes its name
+    from the practice sessions linked to a course, called in French "Travaux Pratiques".
+  doc_url: https://travo-cr.gitlab.io/travo/
+  dev_url: https://travo-cr.gitlab.io/travo/contributing.html
+
+extra:
```
Comment (xhochy) [inline recipes/travo/meta.yaml:86]:
```suggestion
extra:
  feedstock-name: travo
```

**Review by xhochy — CHANGES_REQUESTED** [review]

**Comment by cmarmo** [comment]
Thanks @xhochy !
I believe I have addressed your comments.

File: recipes/travo/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,98 @@
...
+
+build:
+  noarch: python
+  number: 0
+  entry_points:
+    - travo = travo.console_scripts:travo
+    - travo-echo-travo-token = travo.console_scripts:travo_echo_travo_token
+
+outputs:
+  - name: travo
```
Comment (xhochy) [inline recipes/travo/meta.yaml:20]: I think you need to re-iterate this for all outputs
```suggestion
  - name: travo
    build:
      noarch: python
```

**Review by xhochy — APPROVED** [review]

**Comment by cmarmo** [comment]
Thanks @xhocky!

**Pull request merged**